catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/mappings/CswRecordMapperFilterVisitor.java

```diff
@@ -609,5 +608,17 @@ public Object visit(Function function, Object extraData) {
 
         }
     }
-}
 
+    private String getDefaultMetacardFieldForPrefixedString(String propertyName) {
+        if (propertyName.contains(":") && !isXpathPropertyName(propertyName)) {
```

Comment:
❓ About the name `getDefaultMetacardFieldForPrefixedString`. Why is it a default metacard? I see `default` used other places, like in the `DefaultCswRecordMap`, but I don't know what is defaulted because it looks like the metacard types are always passed into the MappedFilterVisitor. Nothing is defaulted.